img/calendar-white.svg

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
:+1: But we should keep it for now until all supported Nextcloud versions ship with that icon.